torch/testing/_internal/common_utils.py

```diff
@@ -1426,111 +1339,37 @@ def _is_dict(self, obj):
     # TODO: default exact_device to True
     def assertEqual(self, x, y, msg: Optional[str] = None, *,
                     atol: Optional[float] = None, rtol: Optional[float] = None,
-                    equal_nan=True, exact_dtype=True, exact_device=False) -> None:
+                    equal_nan=True, exact_dtype=True, exact_device=False, exact_stride=False,
+                    exact_is_coalesced=False) -> None:
         assert (atol is None) == (rtol is None), "If one of atol or rtol is specified, then the other must be too"
```

Comment:
Do we still need to require this, or can assert_close() handle this behavior?

Reply:
Since we need to handle the tolerances [here](https://github.com/pytorch/pytorch/blob/0db4378fc182037da2ab25f45eeab3b266e2fad5/torch/testing/_internal/common_utils.py#L1507-L1512), we need to keep it or branch there.